numpy/core/src/multiarray/nditer_constr.c

```diff
@@ -1410,6 +1410,38 @@ check_mask_for_writemasked_reduction(NpyIter *iter, int iop)
     return 1;
 }
 
+/*
+ * Check whether a reduction is OK based on the flags and the operand being
+ * readwrite. This path is deprecated, since usually only specific axes
+ * should be reduced. If axes are specified explicitely, the flag is
+ * unnecessary.
+ */
+static int
+npyiter_check_reduce_ok_and_set_flags(
+        NpyIter *iter, npy_uint32 flags, npyiter_opitflags *op_itflags) {
```

Comment:
I think our style is usually closer to
```suggestion
        NpyIter *iter, npy_uint32 flags, npyiter_opitflags *op_itflags)
{
```